src/cocoa_window.m

```diff
@@ -682,20 +682,18 @@ - (void)setMarkedText:(id)string
 
     NSString* markedTextString = markedText.string;
 
-    NSUInteger length = [markedTextString length];
-    int ctext = window->ctext;
-    while (ctext < length + 1)
-    {
-        ctext = ctext == 0 ? 1 : ctext * 2;
-    }
-    if (ctext != window->ctext)
+    NSUInteger textLen = [markedTextString length];
+    int textAllcLen = window->preeditLengthAllocated;
```

Comment:
```suggestion
    int textBufferCount = window->preeditBufferCount;
```

* `Allc` is hard to understand since it's too omitted.
* I prefer `Buffer` rather than `Allocated`.
* `Length` or `Count` or something similar suffix should be used for variables which stores array length
* GLFW seems prefer `Count` rather than `Length`

Reply:
Thanks! `textBufferCount` is definitely easier to understand. I will fix it.

> `Allc` is hard to understand since it's too omitted.

I'm interested in this as a general issue, so I'd like to ask the following.

`textAllocatedLength` or `textLengthAllocated` seemed like an unnecessarily long name for this local variable
since we could find the definition immediately and could tell its meaning.

An unnecessarily long name makes the code more difficult to understand, so I thought some commission was good in this case.

I think it's best to use `textBufferCount` in this case, but what about this idea itself?